### ISSUE TITLE
Doc comments get broken

### ISSUE THREAD

**Issue by martin-t** [immunant/c2rust#175]
```c
/// Does nothing
void test1() {}

/** Stilll nothing
 */
void test2() {}
```

results in

```rust
// / Does nothing
#[no_mangle]
pub unsafe extern "C" fn test1() { }
/* * Stilll nothing
 */
#[no_mangle]
pub unsafe extern "C" fn test2() { }
```

There's an extra space after the comment's first 2 chars.

Tested on the website and latest git (d1bbe9f).

**Comment by rinon** [comment]
Yes, this escaping of doc comments was intentional as not all doc comments in all positions are valid in rust (see #61). I'll keep this open as we can probably improve doc comment handling by parsing the comment and position and allowing valid combinations.
